doc/lsst.analysis.ap/tasks/lsst.analysis.ap.PlotImageSubtractionCutoutsTask.rst

```diff
@@ -131,3 +131,39 @@ Configuration fields
 ====================
 
 .. lsst-task-config-fields:: lsst.analysis.ap.PlotImageSubtractionCutoutsTask
+
+.. py:currentmodule:: lsst.analysis.ap
+
+Generating Image Cutouts
+========================
+
+To generate image cutouts using the ``plotImageSubtractionCutouts`` task from ``analysis_ap``, use the
+code below.
+
+Note: Use the ``diaSources`` from the ``goodSeeingDiff_diaSrcTable`` which have the required dataframe format
+for this task.
+
+.. code-block:: python
+
+    import lsst.daf.butler as dafButler
+    from lsst.analysis.ap import plotImageSubtractionCutouts
+
+    repo = '/repo/main'
+    collections = '/path/to/your/collection'
+    butler = dafButler.Butler(repo, collections=collections)
+    where = "instrument='LSSTComCam' and skymap='lsst_cells_v1' and visit=2024120600049"
```

Comment:
Since this is an example, I think it is important to point out that the outer double quotes and inner single quotes are required. I believe if you switch those (e.g. `'instrument="LSSTComCam"`... it will raise an error because that is not valid SQL syntax, even though Python doesn't care.